.mailmap

```diff
@@ -352,11 +352,13 @@ Austin Palmer <ap4000@nyu.edu> austin <ap4000@nyu.edu>
 Avi Shrivastava <shrivastavaavi123@gmail.com> avi <shrivastavaavi123@gmail.com>
 Avichal Dayal <avichal.dayal@gmail.com>
 Ayodeji Ige <ayodeji18@outlook.com>
+Ayush Aryan <ayush.aryan71@gmail.com>
 Ayush Bisht <bisht.ayush2001@gmail.com> Ayush <bisht.ayush2001@gmail.com>
 Ayush Bisht <bisht.ayush2001@gmail.com> ayushbisht2001 <61404154+ayushbisht2001@users.noreply.github.com>
 Ayush Kumar <ayushk7102@gmail.com> Ayush Kumar <65803868+ayushk7102@users.noreply.github.com>
 Ayush Malik <ayushmalik779@gmail.com> Ayush-Malik <ayushmalik779@gmail.com>
 Ayush Shridhar <ayush.shridhar1999@gmail.com>
+Ayush aryan <75092320+ayush3781@users.noreply.github.com>
```

Comment:
I guess we need to make the instructions clearer somehow but you should not add separate entries to the .mailmap file:
https://github.com/sympy/sympy/blob/4fb6b42dd0941f05178aa570edf9ef61162ce960/.mailmap#L138-L148
I'm going to change it so that all of your commits are recorded under:
```
Ayush Aryan <ayush.aryan71@gmail.com>
```

Reply:
I'm changing this in #24775. Please comment there if you would like a different name/email.